### ISSUE TITLE
can not find input module:

### ISSUE THREAD

**Issue by xblox** [nexe/nexe#274]
when running on a windows net share:

```js
nexe -i "Y:\\server\\nodejs\\nxappmain\\server-nexe-main.js" -r 6.9.1 -o ./dist/windows/server.exe -t "Y:\\server\\nexeTemp_windows"
module.js:471
    throw err;
    ^

Error: Cannot find module 'Y:\server\nodejs\**Y:\server\**nodejs\nxappmain\server-nexe-main.js'
    at Function.Module._resolveFilename (module.js:469:15)
```

we had this bug already and you did something about just that it still doesn't work: 

Please see for the correct behaviour in https://github.com/gbaumgart/nexe/blob/master/bin/nexe#L64

This one will check whether the input string already exists and therefore you don't need to join the path. 

If you don't check the input string, then you get also other artifacts for other paths btw.

**Comment by gbaumgart** [comment]
unfortunately, i can't make a PR because i am stuck to that version in other projects.

**Comment by calebboyd** [comment]
fixed in #308
